s3torchconnector/src/s3torchconnector/s3checkpoint.py

```diff
@@ -16,9 +16,10 @@ class S3Checkpoint:
     torch.load, and torch.save.
     """
 
-    def __init__(self, region: str):
+    def __init__(self, region: str, endpoint: str = ""):
```

Comment:
Same here for default `None` and other occurances

Reply:
Changed the default value of endpoint parameter to None instead of empty string